simple_history/models.py

```diff
@@ -265,10 +252,25 @@ def get_history_change_reason(self):
         else:
             history_id_field = models.AutoField(primary_key=True)
 
+        if self.history_change_reason_field:
+            # User specific field from init
+            history_change_reason_field = self.history_change_reason_field
+            history_change_reason_field.null = True
```

Comment:
I don't think we have to do this. If someone wants to make their change_reason nullable with an override, I don't see a reason not to allow it.